.github/workflows/linux-tensorflow.yml

```diff
@@ -0,0 +1,102 @@
+name: Linux Tensorflow Build
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2 
+      with:
+        # Needed for commands that depend on git tags
+        fetch-depth: 0
+        ref: ${{ github.event.pull_request.head.sha }}
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.15.5' 
+    - name: Cache go modules
+      id: cache-go-mod
+      uses: actions/cache@v2.1.5
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+    - name: Cache ffmpeg
+      id: cache-ffmpeg
+      uses: actions/cache@v2.1.5
+      with:
+        path: ~/compiled
+        key: ${{ runner.os }}-ffmpeg-tensorflow-${{ hashFiles('**/install_ffmpeg.sh') }}
+        restore-keys: |
+          ${{ runner.os }}-ffmpeg-tensorflow
+    - name: Cache binaries
+      uses: actions/cache@v2.1.5
+      with:
+        path: ~/build
+        key: ${{ runner.os }}-binaries-tensorflow-${{ github.sha }}
+    - name: Install dependencies
+      run: |
+        sudo apt-get update \
+          && sudo apt-get install -y software-properties-common curl apt-transport-https \
+          && sudo apt-key adv --keyserver keyserver.ubuntu.com --recv 15CF4D18AF4F7421 \
+          && sudo add-apt-repository "deb [arch=amd64] http://apt.llvm.org/xenial/ llvm-toolchain-xenial-8 main" \
+          && sudo apt-get update \
+          && sudo apt-get -y install clang-8 clang-tools-8 build-essential pkg-config autoconf gnutls-dev git python
+
+        sudo update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-8 30 \
+          && sudo update-alternatives --install /usr/bin/clang clang /usr/bin/clang-8 30
+
+        LIBTENSORFLOW_VERSION=2.3.0 \
+          && curl -LO https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-gpu-linux-x86_64-${LIBTENSORFLOW_VERSION}.tar.gz \
+          && sudo tar -C /usr/local -xzf libtensorflow-gpu-linux-x86_64-${LIBTENSORFLOW_VERSION}.tar.gz \
+          && sudo ldconfig
+    - name: Install go modules
+      if: steps.cache-go-mod.outputs.cache-hit != 'true'
+      run: go mod download
+    - name: Install ffmpeg
+      if: steps.cache-ffmpeg.outputs.cache-hit != 'true'
+      run: BUILD_TAGS=experimental ./install_ffmpeg.sh
```

Comment:
Decided to explicitly enable the experimental tag rather than using the `ci_env.sh` script as we'd *always* want a tensorflow build here, even if it's a major release.

Reply:
So, this GH action will always build Tensorflow enabled binaries. But, the scene classification capability will only be enabled in the binary for non-semver tagged releases (i.e. an orchestrator would not advertise the capability to a broadcaster even though the binary itself has Tensorflow support). Seems reasonable.